src/shared/services/form/form-value.service.ts

```diff
@@ -69,6 +69,12 @@ export class FormValueService {
     Object.keys(cloneForm['data']).forEach((key) => {
       if (caseFields.findIndex((element) => element.id === key) < 0) {
         delete cloneForm['data'][key];
+      } else {
```

Comment:
why is this necessary can you explain please?

Reply:
removed